src/main/java/sg/ncl/MainController.java

```diff
@@ -318,9 +318,10 @@ public String testbedNodesStatus(Model model) throws IOException {
         Map<String, List<Map<String, String>>> nodesStatus = getNodesStatus();
         Map<String, Map<String, Long>> nodesStatusCount = new HashMap<>();
 
-        // loop through each of the machine type
-        // tabulate the different nodes type
-        // count the number of different nodes status, e.g. SYSTEMX = { FREE = 10, IN_USE = 11, ... }
+        /* loop through each of the machine type
+           tabulate the different nodes type
+           count the number of different nodes status, e.g. SYSTEMX = { FREE = 10, IN_USE = 11, ... }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)